# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/raku-librarymake/hotfix-ld.patch
# Copyright (C) 2021 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- P6-LibraryMake-7aae514/lib/LibraryMake.rakumod.vanilla	2021-07-11 15:24:19.191000000 +0200
+++ P6-LibraryMake-7aae514/lib/LibraryMake.rakumod	2021-07-11 15:26:16.309000000 +0200
@@ -184,7 +184,12 @@
     }
 
     for %vars.kv -> $k, $v {
-      %vars{$k} [R//]= %*ENV{$k};
+      if %*ENV<SDECFG_VERBOSE>:exists and $k eq 'LD' {
+          # Inside T2 %LD% should be set to $CC
+          %vars{$k} [R//]= %*ENV<CC>;
+      } else {
+          %vars{$k} [R//]= %*ENV{$k};
+      }
     }
 
     return %vars;
